src/RulesExtension.php

```diff
@@ -4,15 +4,59 @@
 
 namespace ProfessionalWiki\Rules;
 
+use MediaWiki\MediaWikiServices;
+use MediaWiki\Message\Message;
+use MediaWiki\Storage\PageUpdater;
+use MediaWiki\Title\Title;
+use MediaWiki\User\User;
+use RuntimeException;
+
 class RulesExtension {
 
 	public const RULES_PAGE_TITLE = 'Rules';
 
+	public const RULES_CONTENT_MODEL = 'rules';
+
+	public const RULES_SYSTEM_USER = 'Rules Bot';
+
+	public const RULES_DEFAULT_CONFIG = '{
```

Comment:
Is there a cleaner way to load a JSON string?

Reply:
I would put this constant on the content class. ... Or actually: put a method on there

```php

class RulesContent {

    public function newDefault() {}

}
```

And then call that from RulesHooks and RulesContentHandler.